src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.html

```diff
@@ -1,5 +1,5 @@
 <xuilib-loading-spinner *ngIf="showSpinner$ | async"></xuilib-loading-spinner>
-
```

Comment:
this should be added to src/work-allocation2/containers/task-list-wrapper/task-list-wrapper.component.html, not src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.html?